custom.example.css

```
@@ -39,20 +46,21 @@
 .image--metadata--exif{}
 .image--metadata--exif--fnumber{}
 .image--metadata--exif--seperator{}
-.image--metadata--location{}
+.frame--layout-splitview .image--metadata--desciption, .frame--layout-splitview-landscape .image--metadata--desciption{}
 .frame--layout-splitview:nth-child(1) .image--metadata{}
 .frame--image-zoom img{}
-.frame--image-zoom-in img{}
-.frame--image-zoom-out img{}
-.frame--image-smart-zoom-out img{}
-.frame--image-smart-zoom-in img{}
+.frame:nth-last-of-type(-n+2) .frame--image-zoom-in img{}
```

Biome

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)